wwwroot/index.html

```diff
@@ -8,6 +8,7 @@
 	<base href="/"/>
 	<link href="css/bootstrap/bootstrap.min.css" rel="stylesheet"/>
 	<link href="css/app.css" rel="stylesheet"/>
+	<link href="css/Navbar.css" rel="stylesheet"/>
```

Comment:
are u sure these are needed ?

Reply:
No it was to learn how to do it and to give an example

app.css and bootstrap.min.css are **required** but Navbar isn't, i think I even deleted it in login-register branch